app/assets/javascripts/discourse/app/widgets/search-menu.js

```diff
@@ -367,7 +367,7 @@ export default createWidget("search-menu", {
       return;
     }
 
-    if (e.key === "A") {
+    if (e.which === 65 /* a */) {
```

Comment:
Could have gone with `e.key === "a"` here, but in tests I can't use the lowercase "a", it would result in: `Error: Must provide a `key` to `triggerKeyEvent` that starts with an uppercase character but you passed `a`.`

So I opted to revert to previous convention and just add a test.

Reply:
Just noting I tried this locally and you can still use `if (e.key === "a") {` for clarity in the code, as long as the test stays the same way with `triggerKeyEvent("#search-term", "keydown", 65);`. 

There is even an ember issue for this with an identical character! Though absolutely no traction:

https://github.com/emberjs/ember-test-helpers/issues/772

Such an oddly specific limitation.

Reply:
Oh, good find, yeah that test suite limitation is weird. 

I'm getting flakiness locally in tests when using `e.key === "a"`, when I run the individual test it works fine but if I run a larger subset, it fails... so I'll stick with the less elegant but tried-and-tested keyCode. 

Thanks for your review!

Reply:
Fair enough, don't want more flakes in there :)